ChangeLog

```diff
@@ -1,3 +1,9 @@
+2015-09-07 Sam Lee <sam.rr.lee@gmail.com>
+
+	* OpenSSL/SSL.py, OpenSSL/test/test_ssl.py: Implemented
+	  the empty ``Connection.state_string()`` function, using
```

Comment:
Can you remove the word "empty", I think it's confusing unless you know the full history of this bug.